test/acceptance/accessibility.py

```diff
@@ -120,12 +120,26 @@ def test_staff_info_panel(self):
         self.staff_area_page.click_staff_toolbar_button("staff-info")
         self._check_a11y(self.staff_area_page)
 
+    def test_staff_grading_panel(self):
+        """
+        Check the accessibility of the "Staff Grading" panel
+        """
+        self.submission_page.visit().submit_response(self.SUBMISSION)
+        self.assertTrue(self.submission_page.has_submitted)
+
+        self.staff_area_page.visit()
+        self.staff_area_page.click_staff_toolbar_button("staff-grading")
+        self._check_a11y(self.staff_area_page)
```

Comment:
Done, it will now expand that section as part of this test and check the whole thing in one fell swoop